<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWndClass</class>
 <widget class="QMainWindow" name="MainWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>774</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kinect Explorer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>482</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="centerWidgetLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="depthImageLabel640x480">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="colorImageLabel640x480">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_InitDirectly"/>
    <addaction name="separator"/>
    <addaction name="action_InitFromXML"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_AboutKinectExplorer"/>
    <addaction name="separator"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_StartKinect"/>
    <addaction name="action_PauseKinect"/>
    <addaction name="action_StopKinect"/>
    <addaction name="separator"/>
    <addaction name="action_KinectSettings"/>
   </widget>
   <widget class="QMenu" name="menuHead_Tracking">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Head Tracking</string>
    </property>
    <addaction name="action_DefaultDepthView"/>
    <addaction name="separator"/>
    <addaction name="action_SimpleHeadTracking"/>
    <addaction name="action_HumanDetection"/>
    <addaction name="action_HumanDetectionWithoutProcessing"/>
   </widget>
   <widget class="QMenu" name="menuHole_Filling">
    <property name="title">
     <string>Hole Filling</string>
    </property>
    <addaction name="action_DefaultNoFilling"/>
    <addaction name="action_NearestNeighborFilling"/>
    <addaction name="action_FMMInpaintFilling"/>
   </widget>
   <widget class="QMenu" name="menuDenoising">
    <property name="title">
     <string>Denoising</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStatus">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuDepth_Map">
     <property name="title">
      <string>Depth Map</string>
     </property>
     <addaction name="action_DepthViewYellow"/>
     <addaction name="action_DepthViewGray"/>
     <addaction name="action_DepthViewRGB"/>
    </widget>
    <addaction name="action_ColorMap"/>
    <addaction name="menuDepth_Map"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStatus"/>
   <addaction name="menuHole_Filling"/>
   <addaction name="menuDenoising"/>
   <addaction name="menuHead_Tracking"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="kinectSettingsDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>680</width>
     <height>123</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>123</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="kinectSettingsDockSubwidget">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1481</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="kinectSettingsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>91</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Kinect Settings</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>111</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <widget class="QRadioButton" name="msSDK">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Microsoft SDK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="openNI">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>OpenNI</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>141</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>Kinect Mode</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="kinectModeComboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>20</y>
           <width>161</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Depth Settings</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="depthSettingsComboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>20</y>
           <width>161</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>RGB Settings</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="rgbSettingsComboBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>91</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Canny Operator Settings</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>171</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QSlider" name="cannyThresholdValueSlider_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="sliderPosition">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="cannyThresholdValueSlider_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="action_InitFromXML">
   <property name="text">
    <string>Init From XML</string>
   </property>
  </action>
  <action name="action_StartKinect">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/icons/start24.ico</normaloff>:/MainWnd/icons/start24.ico</iconset>
   </property>
   <property name="text">
    <string>Start Kinect</string>
   </property>
  </action>
  <action name="action_PauseKinect">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/icons/pause24.ico</normaloff>:/MainWnd/icons/pause24.ico</iconset>
   </property>
   <property name="text">
    <string>Pause Kinect</string>
   </property>
  </action>
  <action name="action_StopKinect">
   <property name="icon">
    <iconset resource="MainWnd.qrc">
     <normaloff>:/MainWnd/icons/stop24.ico</normaloff>:/MainWnd/icons/stop24.ico</iconset>
   </property>
   <property name="text">
    <string>Stop Kinect</string>
   </property>
  </action>
  <action name="action_InitDirectly">
   <property name="text">
    <string>Init Directly</string>
   </property>
  </action>
  <action name="action_ColorMap">
   <property name="text">
    <string>Color Map</string>
   </property>
  </action>
  <action name="actionSkeleton_Map">
   <property name="text">
    <string>Skeleton Map</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_AboutKinectExplorer">
   <property name="text">
    <string>About Kinect Explorer</string>
   </property>
  </action>
  <action name="action_KinectSettings">
   <property name="text">
    <string>Kinect Settings</string>
   </property>
  </action>
  <action name="action_ColorDepthMap">
   <property name="text">
    <string>Color &amp; Depth Map</string>
   </property>
  </action>
  <action name="action_SimpleHeadTracking">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simple Head Tracking</string>
   </property>
  </action>
  <action name="action_HumanDetection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Human Detection</string>
   </property>
  </action>
  <action name="actionDefault_Depth_View_Yellow">
   <property name="text">
    <string>Default Depth View ( Yellow )</string>
   </property>
  </action>
  <action name="action_DefaultDepthView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default Depth View</string>
   </property>
  </action>
  <action name="action_HumanDetectionWithoutProcessing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Human Detection Without Processing</string>
   </property>
  </action>
  <action name="action_DefaultNoFilling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default (No Filling)</string>
   </property>
  </action>
  <action name="action_NearestNeighborFilling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nearest Neighbor Filling</string>
   </property>
  </action>
  <action name="action_FMMInpaintFilling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FMM Inpainting</string>
   </property>
  </action>
  <action name="action_DepthViewYellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="action_DepthViewRGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="action_DepthViewGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWnd.qrc"/>
 </resources>
 <connections/>
</ui>
